test/fixtures/cluster-preload.js

```diff
@@ -1,3 +1,15 @@
+var assert = require('assert');
+
+// https://github.com/nodejs/io.js/issues/1803
+// this module is used as a preload module. It should have a parent with the
+// module search paths initialized from the current working directory
+assert.ok(module.parent);
+var expectedPaths = require('module')._nodeModulePaths(process.cwd());
+assert.ok(module.parent.paths.length === expectedPaths.length &&
+  module.parent.paths.every(function(e,i) {
```

Comment:
Space after the `,`. I think `module` should line up with `module` on the previous line.